tests/cpp/multidevice.cpp

```diff
@@ -170,12 +166,12 @@ void PipelineTest::execute() {
   if (error_msg != "") {
     GTEST_SKIP() << error_msg;
   }
+  outputs = runtime->runWithInput(inputs);
   if (debug_print) {
     if (!communicator->deviceId()) {
       runtime->print();
     }
   }
-  outputs = runtime->runWithInput(inputs);
 
   if (debug_print) {
```

Comment:
```suggestion
```

Reply:
The change suggestion doesn't work very well in this case. I meant to merge the two `if (debug_print)`.